active/PagerControl/PagerControl.md

```diff
@@ -0,0 +1,356 @@
+| NextButton | * Button displaying text and/or glyph indicating that the user may navigate to the next index. <br> * Automatically disabled when at last index. <br> * Can be set to not be visible when at the last index. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760426-54474780-fc79-11e8-9446-ee2886941a24.png) |
+| PreviousButton | * Button displaying text and/or glyph indicating that the user may navigate to the previous index. <br> * Automatically disabled when at first index. <br> * Can be set to not be visible when at the first index.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760443-62956380-fc79-11e8-8f93-562a92fde666.png) |
+| Ellipsis | * Inactive Button, often reading "...", used between indexes and before or after the first/last index to indicate an accessible but omitted range of indexes. <br> * MaxBefore and MaxAfter properties can be used to set  how many indices appear between the current page and the ellipsis before/after it. <br> * Visibility of the first/last index can be disabled. <br> * Only visible when using button panel as the display mode.  <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760513-85277c80-fc79-11e8-926e-3453ea29b0a3.png) |
+| PrefixText | * Text displayed before the editable ComboBox indexing component. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760540-93759880-fc79-11e8-9c4f-3c3222b95a13.png) |
+| NumberOfPages | * When a total number of indices (N) is given, this suffix string will appear after the editable ComboBox indexing component and read "of N". Localization will put "N" where it should be in a given language. <br><br> ![image](https://user-images.githubusercontent.com/16964652/49760557-9a9ca680-fc79-11e8-943f-3425d993fdeb.png) |
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+enum PagerDisplayMode
+{
+    Auto,
+    ComboBox,
+    NumberBox,
+    ButtonPanel,
+};
+
+
+enum ButtonVisibilityMode
+{
+    Auto,
+    AlwaysVisible,
+    HiddenOnLast,
+    None,
+};
+
+runtimeclass PagerControlPageChangedEventArgs
+{
+    Integer CurrentPage{get; }:
+}
+
+runtimeclass PagerControl
+{
+    PagerControl();
+
+    PagerDisplayMode Display;
+    
+    Integer NumberOfPages;
+    
+    ButtonVisibilityMode FirstButtonVisibility;
+    ButtonVisibilityMode PreviousButtonVisibility;
+    ButtonVisibilityMode NextButtonVisibility;
+    ButtonVisibilityMode LastButtonVisibility;
+
+    FirstButtonCommand="FirstButtonPressedEvent"
+    PreviousButtonCommand="PreviousButtonPressedEvent"
+    NextButtonCommand="NextButtonPressedEvent"
+    LastButtonCommand="LastButtonPressedEvent"
+    PagerInputCommand="PagerInputEvent"
+    
+    IconSource FirstButtonGlyph;
```

Comment:
A glyph is always a string today. I think all of our other IconSource properties have an IconSource suffix

Reply:
We decided to remove these and just keep the button style.